modules/core/src/main/java/org/apache/ignite/internal/processors/cache/binary/BinaryMetadataFileStore.java

```diff
@@ -54,6 +63,12 @@
     /** */
     private final IgniteLogger log;
 
+    /** */
+    private BinaryMetadataAsyncWriter writer;
+
+    /** */
+    private final ConcurrentHashMap<WriteOpSync, GridFutureAdapter> writeOpFutures = new ConcurrentHashMap<>();
```

Comment:
Type of the field can be changed to ConcurrentMap

Reply:
Done.